### PR TITLE
fix(styles): Button-group checkbox/radio are visible

### LINKED ISSUE #2084: Styles: Visual regression tests for the button group
Write visual regression tests for the [button group component](https://next.design-system.post.ch/?path=/story/components-text-input--default).

Create a storybook page similar to [the button test page](https://next.design-system.post.ch/?path=/story/hidden-button-snapshot-test--page) that shows all variants on one page. In the snapshot test, navigate to this page and take one snapshot of all variants at once.

```[tasklist]
### Properties to test
- [x] elements
- [x] Size
- [x] Checked state
- [ ] Pseudo-class
```

### LINKED ISSUE #2131: Bug(documentation): button group input and labels are displayed seperate
The Button group component does not look correct at the moment.

image

and

image

### PR DESCRIPTION
**Pull request by imagoiq**
### What

- Fix over https://github.com/swisspost/design-system/pull/2064/files#diff-ae122d73c2e914ec29f656456684332a7282fb139cd8c3e3837a0a0b37d9b604L5 which removes Bootstrap Form-Check and includes `btn-check` styles.
- Create visual regression tests for button-group

Closes #2131 #2084 

#### Before

image

### CODE REVIEW

File: packages/documentation/src/stories/components/button-group/button-group.stories.ts [hunk 1]
```diff
@@ -144,23 +144,24 @@ type Story = StoryObj;
...
   const position = index + 1;
-  const id = `btngroup${context.name}_${position}`;
+  const id = `btngroup_${context.name}_${position}`;
+  const name = `btngroup_${context.name}`;
   const label = args[`label_${position}`];
 
   switch (args.element) {
     case 'checkbox': {
-      const isSelected = args.selected?.includes(position) ?? false;
+      const isSelected = position === args.selected;
```
Comment (alizedebray) [inline packages/documentation/src/stories/components/button-group/button-group.stories.ts:153]: This does not work with checkboxes

**Review by alizedebray — CHANGES_REQUESTED** [review]
A focus shadow is missing for checkbox and radio buttons

**Comment by imagoiq** [comment]
> A focus shadow is missing for checkbox and radio buttons

Yes this is the ticket: https://github.com/swisspost/design-system/issues/2134

File: packages/documentation/src/stories/components/button-group/button-group.stories.ts (hunk 1 above)
Comment (imagoiq) [reply packages/documentation/src/stories/components/button-group/button-group.stories.ts]: Fixed!

**Review by alizedebray — APPROVED** [review]

**Pull request merged**